docs/src/main/asciidoc/_admin-guide/Management_resources.adoc

```diff
@@ -285,6 +285,62 @@ See <<descriptions,Descriptions>>
 below for how to learn more about the attributes a particular resource
 exposes.
 
+[[overridden-attribute-value]]
+=== Override an Attribute Value with an Environment Variable
+
+It is possible to override the value of any simple attribute by providing an environment variable with a name
+that maps to the attribute (and its resource).
+
+[NOTE]
```

Comment:
This not does not render correctly in 
https://github.com/wildfly/wildfly/blob/c5b89a2567849cbf3d8334d9655add4f478748b8/docs/src/main/asciidoc/_admin-guide/Management_resources.adoc

It leaves the equals signs in the note. I think we need to use 4 equal signs here

Reply:
fixed, thanks!